databricks/koalas/missing/series.py

```diff
@@ -156,6 +156,10 @@ class _MissingPandasLikeSeries(object):
     real = unsupported_property(
         'real',
         reason="If you want to collect your data as an NumPy array, use 'to_numpy()' instead.")
+    nbytes = unsupported_property(
+        'nbytes',
+        reason="Since nbytes trigger whole computation to get the size, which is pretty "
```

Comment:
You can type it like:

```
'nbytes' requires to compute whole dataset. You can calculate manually it, with its 'itemsize', 
by explicitly executing its count. Use Spark's web UI to monitor disk and memory usage of 
your application in general.
```

Reply:
Thank you for help. i've fixed comment and commit it. :)